### PR TITLE
drift: Exit with nonzero code if drift detected

### PR DESCRIPTION
**Pull request by disq**
Exit with exit code 1 if any of these are detected:
- drifted resources
- extra resources
- missing resources

### CODE REVIEW

File: cmd/drift.go [hunk 1]
```diff
@@ -53,12 +54,20 @@ var (
...
 
-			return c.CallModule(ctx, console.ModuleCallRequest{
+			err = c.CallModule(ctx, console.ModuleCallRequest{
 				Name:          driftModuleID,
 				Params:        driftParams,
 				ModConfigPath: driftConfigPath,
 				OutputPath:    driftOutputPath,
 			})
+			if err != nil {
+				if ee, ok := err.(*console.ExitCodeError); ok {
```
Comment (disq) [inline cmd/drift.go:64]: This code could be simplified if we decide we only need just one nonzero exit code.

**Comment by spangenberg** [comment]
@disq can you add the handleError from #257, guess you can and should still use the os.Exit as you have it for specific codes.

**Comment by disq** [comment]
> @disq can you add the handleError from #257, guess you can and should still use the os.Exit as you have it for specific codes.

Already added 4 minutes before your comment 👀

**Review by spangenberg — APPROVED** [review]
LGTM

**Review by roneli — APPROVED** [review]

**Pull request merged**